client/qiskit_serverless/core/clients/serverless_client.py

```diff
@@ -570,6 +580,22 @@ def __init__(
             channel: identifies the method to use to authenticate the user
         """
         token = token or QiskitRuntimeService(name=name).active_account().get("token")
+        channel = channel or QiskitRuntimeService(name=name).active_account().get(
+            "channel"
+        )
+        try:
+            channel_enum = Channel(channel)
+        except ValueError as error:
+            raise QiskitServerlessException(
+                "Your channel value is not correct. Use one of the available channels: "
+                f"{Channel.LOCAL.value}, {Channel.IBM_QUANTUM.value}, {Channel.IBM_CLOUD.value}"
```

Comment:
The error message should show the new iqp channel too (the line is probably going to be too long after the suggestion, sorry)

```suggestion
                "Your channel value is not correct. Use one of the available channels: "
                f"{Channel.LOCAL.value}, {Channel.IBM_QUANTUM_PLATFORM.value}, {Channel.IBM_QUANTUM.value},  {Channel.IBM_CLOUD.value}"
```

Reply:
done! [9d385a0](https://github.com/Qiskit/qiskit-serverless/pull/1657/commits/9d385a0eadadfb86bebd81b793dcbc229fcf2cdf)[9d385a0](https://github.com/Qiskit/qiskit-serverless/pull/1657/commits/9d385a0eadadfb86bebd81b793dcbc229fcf2cdf)